doc/install/install-ceph-gateway.rst

```diff
@@ -510,24 +510,24 @@ Execute the following steps:
 
 #. Add the following contents to the file::
 
-    import boto
```

Comment:
Couldn't really stand it as well so I looked it up. `connect_s3` is defined in the `__init__.py` of top level of the boto package, which gets read even when you import a subpackage like `boto.s3.connection`